src/TodoListItem.jsx

```diff
@@ -0,0 +1,11 @@
+const TodoListItem = (props) => {
+    const { todo } = props
```

Comment:
This works, but can be avoided by destructing it directly in the function parameters:

```
const TodoListItem = ({todo}) => {}
```

Reply:
Done